tiledb/api/Cargo.toml

```diff
@@ -15,10 +15,12 @@ proptest-derive = { version = "0.4.0", optional = true }
 serde = { version = "1.0.136", features = ["derive"] }
 serde_json = { workspace = true }
 thiserror = "1.0.58"
+tiledb-proc-macro = { workspace = true }
 tiledb-sys = { workspace = true }
+tiledb-utils = { workspace = true, features = ["serde_json"] }
 
 [dev-dependencies]
-tempdir = "0.3.7"
+tempfile = "3"
```

Comment:
I'm vaguely curious on this change. Is it just that you needed a temporary file, and tmpfile is tempdir++?

Reply:
If the assertion fails then there is a panic.  The old crate's `Tempfile` cleaned up in a regular `Drop` but not in a panic.  The new crate also cleans up if there is a panic.